client/src/shapes/shapes/rect/hitbox.ts

```diff
@@ -19,19 +20,20 @@ export const rectHitbox = (rectangle: RectSchema) => (point: Coordinate) => {
     width: normalizedWidth,
     height: normalizedHeight,
   } = normalizeBoundingBox({ at, width, height });
-
   const centerX = normalizedAt.x + normalizedWidth / 2;
   const centerY = normalizedAt.y + normalizedHeight / 2;
-
-  const strokeWidth = stroke?.lineWidth ?? 0;
+  const strokeWidth = stroke?.lineWidth || STROKE_DEFAULTS.stroke.lineWidth;
   const localPoint = rotatePoint(point, { x: centerX, y: centerY }, -rotation);
-
   const { x, y } = {
     x: centerX - normalizedWidth / 2,
     y: centerY - normalizedHeight / 2,
   };
 
-  if (borderRadius === 0) {
+  if (
+    borderRadius === undefined ||
```

Comment:
border radius cannot ever be undefined. If it is thats an invalid type that should be caught at the type level